python/test/pyJointAxis_TEST.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+from ignition.math import Pose3d, Vector3d
+from sdformat import JointAxis, Error
+import math
+import unittest
+
+
+class JointAxisTEST(unittest.TestCase):
+
+  def test_default_construction(self):
```

Comment:
it's probably a problem with all the bindings, but it occurs to me that pep8 wants 4 spaces for indentation?

Reply:
Yes, pep8 uses 4 spaces. You can run `black` to reformat it.

Reply:
https://github.com/ignitionrobotics/sdformat/pull/960/commits/1f6903ff1ac2b7f7217284c2ed19337ae155562d